### PR TITLE
Calculator interaction milestone 5

### PR DESCRIPTION
**Pull request by Salvador-ON**
-[x] I initialized my state object with total, next and operation properties. In my case I designed my  App using Stateless Functional Component. So in my case, I create a hook to manage it use Using React.useState
-[x] Implement the handleClick function which takes buttonName
-[x] Pass the total or next value to Display
-[x] Pass the clickHandler property to each button in the panel.
-[x] Use the onClick event to attach the handleClick event to the button
-[x] Implement the handleClick function

live demo:

https://react-calculator-so.herokuapp.com

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    result = Big(ValueOne).times(Big(ValueTwo));
+  }
+
+  if (opt === '÷') {
+    if (ValueTwo === '0') {
+      result = '0';
+    } else {
+      result = Big(ValueOne).div(ValueTwo);
+    }
+  }
```
Comment (shloch) [inline src/logic/operate.js]: If `ValueTwo` is ZERO, you should return `NaN` or `Infinity` or `zero division error`
It's not zero......

**Review by shloch — COMMENTED** [review]
## Hello @Salvador-ON 

Great job
calc2

Tested everything...works very good.....
Take a look at my comment below, you might bring this small modification before Merging

**Review by shloch — APPROVED** [review]
## You get a thumbs up .... GREAT JOB !

File: src/logic/operate.js (hunk 1 above)
Comment (Salvador-ON) [reply src/logic/operate.js]: thanks, Shey I will make the change and proceed to merge

**Pull request merged**